### PR TITLE
Gulp Example

### PR DESCRIPTION
**Pull request by hironroy**
Hi,

Thanks for the great project! I heard about the project from @rmurphey appearing on Developer Tea.

We use gulp, and I've setup my project to build our scoutfile with it. Since there isn't currently a gulp example, I added one that mirrors the functionality in the grunt example.

Thanks

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
     "scoutfile": "git+ssh://git@github.com:bazaarvoice/scoutfile.git#master"
   },
   "devDependencies": {
-    "grunt": "^0.4.5"
+    "grunt": "^0.4.5",
+    "gulp": "^3.8.11",
+    "scoutfile": "git://github.com/bazaarvoice/scoutfile.git#master",
```
Comment (lawnsea) [inline example/package.json]: Is this necessary? `scoutfile` appears in `dependencies`.

File: example/gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+var gulp = require('gulp'),
+    scoutfile = require('scoutfile'),
```
Comment (lawnsea) [inline example/gulpfile.js]: Please re-indent at 2 spaces per level.

**Comment by lawnsea** [comment]
This is great! Thanks very much.

I have two small nits, but otherwise am :+1: to land this.

**Comment by hironroy** [comment]
I've made the two changes you asked for and rebased them into the commit a255ba75e762ac24efe452bf5cdf0ad2b4a9eda2 (aka. my fork head)

Let me know if there's anything else. Thanks!

**Comment by lawnsea** [comment]
Landed. Thanks!

**Pull request closed without merging**